fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirSchemaGenerator.java

```diff
@@ -6,43 +6,7 @@
 
 package com.ibm.fhir.schema.control;
 
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.CODE_SYSTEMS;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.CODE_SYSTEM_ID;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.CODE_SYSTEM_NAME;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.DATE_END;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.DATE_START;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.DATE_VALUES;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.DATE_VALUE_DROPPED_COLUMN;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.FHIR_REF_SEQUENCE;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.FHIR_SEQUENCE;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.FK;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.IDX;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.LOGICAL_ID;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.LOGICAL_ID_BYTES;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.LOGICAL_RESOURCES;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.LOGICAL_RESOURCE_ID;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.MAX_SEARCH_STRING_BYTES;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.MAX_TOKEN_VALUE_BYTES;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.MT_ID;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.PARAMETER_NAME;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.PARAMETER_NAMES;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.PARAMETER_NAME_ID;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.RESOURCE_TYPE;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.RESOURCE_TYPES;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.RESOURCE_TYPE_ID;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.STR_VALUE;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.STR_VALUES;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.STR_VALUE_LCASE;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.TENANTS;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.TENANT_HASH;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.TENANT_KEYS;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.TENANT_KEY_ID;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.TENANT_NAME;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.TENANT_SALT;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.TENANT_SEQUENCE;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.TENANT_STATUS;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.TOKEN_VALUE;
-import static com.ibm.fhir.schema.control.FhirSchemaConstants.TOKEN_VALUES;
+import static com.ibm.fhir.schema.control.FhirSchemaConstants.*;
```

Comment:
I've got my eclipse set up to expand this on every save.  Any tips/tricks for selectively avoiding that?

Reply:
try `//@formatter:off` and then `//@formatter:on`

Reply:
In this case I like the .* because it ensures I always have all the constants available when I'm typing new code.